### DISCUSSION TITLE
Classification of leading, next-to-leading

### DISCUSSION THREAD

**Discussion by randomizer6542** [MadAnalysis/madanalysis5#196] [Q&A]
Hi, I wanted you to clarify something for me, suppose i classify my leptons as l[1],l[2] and l[3]
what parameter does the madanalysis use to determine if it is leading or next-to-leading and so on, I know transverse momentum is the default one. What I want to know really is at what value of MT is a particle considered leading or next-to-leading and so on, is there any way to know that?

**Comment by BFuks** [comment]
Dear randomized user,

I am not sure to have understood the question. The ordering is performed through the pT variable by default, and not through the transverse mass. If the particle mass is small compared with the typical pT of the objects in the simulated events, the two are however equivalent.

Cheers,

Benjamin

**Reply by randomizer6542** [reply]
Let me clarify, suppose I ask the ma5 to plot l+[1], then the ma5 will plot the leading l+ events, right?
what I want to know is that what makes a particle/event leading, next-to-leading or sub-leading and so on?

**Comment by BFuks** [comment]
Hi Random,

> Let me clarify, suppose I ask the ma5 to plot l+[1], then the ma5 will plot the leading l+ events, right?
> 
Yes. That’s correct.

> what I want to know is that what makes a particle/event leading, next-to-leading or sub-leading and so on?
> 
By default, this consists of its transverse momentum. But this can be changed, as noted in the manual.

Cheers,

Benjamin
